presidio-image-redactor/Pipfile

```diff
@@ -12,6 +12,8 @@ pydicom = ">=2.3.0"
 pypng = ">=0.20220715.0"
 matplotlib = "==3.6.2"
 typing-extensions = "*"
+opencv-python = ">=4.5.0"
```

Comment:
opencv-python needs to be added to `setup.py` too

Reply:
Updated `setup.py` and `Pipefile.lock`

Reply:
Thanks!